### PR TITLE
Show Pending Document Flags when HideFromPortal On

### PR DESCRIPTION
**Pull request by dhaselhan**

### CODE REVIEW

**Review by trslater — APPROVED** [review]
LGTM 🚀

File: alcs-frontend/src/app/features/application/decision/decision-v2/decision-documents/decision-documents.component.ts [hunk 1]
```diff
@@ -37,19 +38,21 @@ export class DecisionDocumentsComponent implements OnInit, OnDestroy {
...
-    private confirmationDialogService: ConfirmationDialogService
+    private confirmationDialogService: ConfirmationDialogService,
   ) {}
 
   ngOnInit(): void {
     if (this.decision && !this.loadData) {
       this.dataSource = new MatTableDataSource(this.decision.documents);
-      this.areDocumentsReleased = !this.decision.isDraft && !!this.decision.date && Date.now() >= this.decision.date;
+      this.areDocumentsReleased =
+        !this.hiddenFromPortal && !this.decision.isDraft && !!this.decision.date && Date.now() >= this.decision.date;
```
Comment (mhuseinov) [inline alcs-frontend/src/app/features/application/decision/decision-v2/decision-documents/decision-documents.component.ts:48]: is decision.date just a date or dateTime?

**Review by mhuseinov — APPROVED** [review]
👍

File: alcs-frontend/src/app/features/application/decision/decision-v2/decision-documents/decision-documents.component.ts (hunk 1 above)
Comment (dhaselhan) [reply alcs-frontend/src/app/features/application/decision/decision-v2/decision-documents/decision-documents.component.ts:48]: Its a number

**Pull request merged**